pkg/debugapi/router.go

```diff
@@ -43,6 +43,9 @@ func (s *server) setupRouting() {
 	router.Handle("/addresses", jsonhttp.MethodHandler{
 		"GET": http.HandlerFunc(s.addressesHandler),
 	})
+	router.Handle("/overlay-address", jsonhttp.MethodHandler{
```

Comment:
why not `/addresses/overlay`?

Reply:
That is a good question. The main reason is that `/addresses` is returning underlay addresses that do not contain overlay address.

Since you asked, maybe I rushed with the implementation. Would it make sense just to add a field `overlay` in `/addresses` response and rename `addresses` field to `underlay` in it?

Reply:
yes, this sounds like a great idea :+1:

Reply:
Thanks, I will change it.

Reply:
Done, @acud ready for another review.